### ISSUE TITLE
Send notifications to all mobile apps

### ISSUE THREAD

**Issue by nicolamori** [immanuelfodor/rocketchat-push-gateway#3]
I'm experimenting with this project in order to setup notifications for a Rocket.Chat instance to serve O(100) users. If I correctly understand it is necessary to manually insert the push token for each Rocket.Chat mobile app in the AppRise configuration file. This is a no-go for me, since it means that I'll have to constantly take care of monitoring the set of connected mobile device and keep the AppRise configuration up-to-date. Is this really the case? If so, is there any way to circumvent this problem and automate the process of registering new clients and removing disconnected ones, so that notifications are automatically sent to all the currently connected mobile apps?

**Comment by immanuelfodor** [comment]
Yes, what you described is how it currently works. Since I had only <4 users, this was not a problem for me. I'm open for PRs if somebody decides to automate some/all of the process but I personally don't have the time nor the resources to achieve this. The main obstackle is that I switched to XMPP some time ago, then switched again lately to Matrix.
